make sure that -lm is included when linking libxc executables
cfr. https://gitlab.com/libxc/libxc/issues/93
author: Kenneth Hoste (HPC-UGent)
--- libxc-4.3.4/CMakeLists.txt.orig	2019-05-21 14:17:21.491272049 +0200
+++ libxc-4.3.4/CMakeLists.txt	2019-05-21 14:16:34.301597402 +0200
@@ -38,7 +38,7 @@
 set(PN ${PROJECT_NAME})
 
 # link -lm only if necessary
-find_package(StandardMathLibraryC)
+set(STANDARD_MATH_LIBRARY "m") #find_package(StandardMathLibraryC)
 # check if cbrt exists and declare HAVE_CBRT if it does
 check_c_source_compiles (
   "#include <math.h>
